syntax = "proto3";
enum MessageRetCode {
   MRC_SUCCESSED = 0;  	//成功
   MRC_FAILED = 1;
   MRC_DISCONNECT = 2;	//连接断开
   MRC_ACCOUNT_EXIST = 3; //账号己存在
	
}

